Tools/WebGPUPlayground/AppPlayground/WebGPUCoordinator.m

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2022 Apple Inc. All rights reserved.
```

Comment:
Not sure how critical this is, but all your licenses still say 2022

Reply:
😳